docs/quickstart/infrastructure.md

```diff
@@ -0,0 +1,127 @@
+<!--
+title: "Infrastructure monitoring with Netdata"
+sidebar_label: "Infrastructure monitoring"
+description: "Build a robust, infinitely-scalable infrastructure monitoring solution with Netdata. "
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/quickstart/infrastructure.md
+-->
+
+# Infrastructure monitoring with Netdata
+
+Together, the Netdata Agent and Netdata Cloud create a powerful, infinitely-scalable infrastructure monitoring solution.
+
+The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly.
+The distributed data model lets you monitor everything at a fraction of the cost compared to centralized metrics
+databases. Netdata Cloud unifies all the metrics from these distributed nodes on customizable, interactive, and
+real-time visualizations.
+
+In this quickstart guide, you'll learn how to see key metrics from all your nodes in one interface and build your first
+dashboard for aggregating like metrics from many distributed nodes. You'll then take a peek into configuring individual
```

Comment:
Suggest clarifying to: "You'll then take a peek into configuring individual nodes and get helpful pointers about collecting all the metrics from every critical application in your infrastructure."